diff -upr a/Tests/L2Tests/L2TestsPlugin/CMakeLists.txt b/Tests/L2Tests/L2TestsPlugin/CMakeLists.txt
--- a/Tests/L2Tests/L2TestsPlugin/CMakeLists.txt	2026-05-08 20:37:30.000000000 +0300
+++ b/Tests/L2Tests/L2TestsPlugin/CMakeLists.txt	2026-05-12 03:11:46.939211395 +0300
@@ -73,6 +73,11 @@ if(PLUGIN_BLUETOOTH)
      target_link_libraries(${MODULE_NAME} PRIVATE ${CMAKE_INSTALL_PREFIX}/lib/libWPEFrameworkL2TestsBT.so)
 endif()
 
+#PLUGIN_ACCOUNT
+if(PLUGIN_ACCOUNT)
+    target_link_libraries(${MODULE_NAME} PRIVATE ${CMAKE_INSTALL_PREFIX}/lib/libWPEFrameworkL2TestsAccount.so)
+endif()
+
 #Added for Contract Verifier tests
 if(CT_VERIFIER_TESTS)
     target_link_libraries(${MODULE_NAME} PRIVATE ${CMAKE_INSTALL_PREFIX}/lib/libWPEFrameworkL2TestsCT.so)
